src/dumbo/src/pdu/ipv4.rs

```diff
@@ -30,7 +30,7 @@ const OPTIONS_OFFSET: usize = 20;
 /// Indicates version 4 of the IP protocol
 pub const IPV4_VERSION: u8 = 0x04;
 /// Default TTL value
-pub const DEFAULT_TTL: u8 = 200;
+pub const DEFAULT_TTL: u8 = 1;
```

Comment:
Some docs about why it's set to 1 would be helpful.

Reply:
Or at least in the commit/PR message

Reply:
Added in the commit message.